addons/udes_priorities/models/priority.py

```diff
@@ -26,17 +26,36 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _compute_outstanding_pickings(self):
+        Picking = self.env["stock.picking"]
+        for record in self:
```

Comment:
Nit: Sorry super super minor one. But can we call this `priority` instead of `record`? As we are looping through a recordset of priorities.

Not a big deal but we try to be as explicit as possible with the variable names. :)